temba/orgs/views.py

```diff
@@ -684,6 +685,22 @@ def clean_import_file(self):
                     json_data["triggers"] = triggers
                     data = json.dumps(json_data)
 
+                if json_data.get("links", []):
+                    links = []
+                    processed_links = {}
+                    for link in json_data["links"]:
+                        full_link = f"{link.get('name')} {link.get('destination')}"
+                        filter_params = {
+                            "org": self.org,
+                            "name": link.get("name"),
+                            "destination": link.get("destination"),
+                            "is_archived": False,
+                        }
+                        if not Link.objects.filter(**filter_params).exists() and not full_link in processed_links:
```

Comment:
Hi @ihor-palii - minor change here, let's make it like this:
```
... and full_link not in processed_links:
```

Reply:
I have changed this line.